src/components/Calculator.style.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+export const CalculatorContainer = styled.section`
+	width: 100%;
+	margin: 0.5rem;
+	@media (min-width: 720px) {
+		margin-right: 5rem;
+	}
+	@media (min-width: 1200px) {
+		margin-right: 10rem;
+	}
+`;
+export const SolidDivider = styled.div`
+	border-left-style: solid;
+	border-left-color: black;
+	border-left-width: 1px;
+	height: 5rem;
+	margin-left: 5rem;
+`;
+export const DashedDivider = styled.div`
```

Comment:
It looks like a dotted style to me in the mock.

Reply:
[Updated](https://github.com/ardriveapp/price-calculator/commit/9268a6cb9e9bebfc75999c9e55fea8a36651668d)